### DISCUSSION TITLE
Change key for presentation mode of windowed pdf viewer?

### DISCUSSION THREAD

**Discussion by octaeder** [texstudio-org/texstudio#3425] [Q&A] [answered]
I know that browsers from Mozilla and Microsoft, notepad++, FreeCommander use F11 key to switch to fullscreen mode. Also main window of txs uses F11 for that, but windowed pdf viewer uses F5. I think it should be reasonable to change it to F11.
Any comments?

**Comment by dbitouze** [comment, marked as answer]
I would prefer to stick on <kbd>F5</kbd>: it is used by other PDF viewers such as Evince, and is more a  “Presentation” than a “Fullscreen” mode.

**Reply by octaeder** [reply]
Thanks for your feedback
